index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Template 1-mobile-main</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&family=Roboto:wght@500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav class="logo">
+        <h1><a href="#">My logo</a></h1>
+        <a><img src="./icons/union.jpg" alt="menu"></a>
```

Comment:
- [ ] You are using `<a>` tag here, a tag is a link, but in your case you have no link, no href attribute. Thus there is no need for `a` tag. I recommend wrapping `humburger` icon with  a `button` tag instead